back/infolica/views/affaire_document.py

```diff
@@ -108,23 +108,23 @@ def download_affaire_document_view(request):
 ###########################################################
 
 
-@view_config(route_name="spch_document", request_method="GET")
+@view_config(route_name="externe_document", request_method="GET")
 def download_affaire_document_view(request):
     """
     Download document for SPCH
     """
-    t_request = request.params["t"] if "t" in request.params else None
-    t_config = request.registry.settings["spch_token"]
+
+    affaire_id = request.params["affaire_id"] if "affaire_id" in request.params else None
+    affaire = request.dbsession.query(Affaire).filter(Affaire.id == affaire_id).first()
+    uuid = request.params["uuid"] if "uuid" in request.params else None
```

Comment:
Don't you need to add a test to check that `affaire_id` is not none (and raise an error if it is the case)?